.github/workflows/unit-tests.yml

```diff
@@ -100,6 +100,12 @@ jobs:
           BIONEMO_DATA_SOURCE: ngc
         run: ./ci/scripts/run_pytest.sh --no-nbval --skip-slow
 
+      - name: Run notebook tests
+        if: ${{ contains(github.event.pull_request.labels.*.name, 'INCLUDE_NOTEBOOKS_TESTS') }}
+        env:
+          BIONEMO_DATA_SOURCE: ngc
+        run: pytest --nbval-lax -p no:python docs/ sub-packages/
```

Comment:
@dorotat-nv, here's how we could do notebook tests on those azure runners triggered by github tag. We just run those nbval tests as a separate step